reconcile/change_owners/change_log_tracking.py

```diff
@@ -28,7 +29,7 @@
 @dataclass
 class ChangeLogItem:
     commit: str
-    created_at: str
+    merged_at: str
```

Comment:
@maorfr this is a breaking change and init from last saved state is failing

```
ChangeLogItem(**i) # type: ignore[arg-type]
^^^^^^^^^^^^^^^^^^
TypeError: ChangeLogItem.__init__() got an unexpected keyword argument 'created_at'
```

why we need to query `merged_at`? `app-interface` build master commit is the merge commit, it's `created_at` is the commit time on master branch, mr `merged_at` timestamp should have less than 1 second diff with that commit time.

Reply:
did you check this assumption?

Reply:
tl;dr not a merge commit ^